tests/network/sriov.go

```diff
@@ -647,6 +645,34 @@ func checkInterfacesInGuest(vmi *v1.VirtualMachineInstance, interfaces []string)
 	}
 }
 
+func isPCIAdsress(token string) bool {
+	return pciAddressRegex.MatchString(token)
+}
+
+func isInterfaceOnRootPCIBridge(vmi *v1.VirtualMachineInstance, iface string) (bool, error) {
+	const (
+		timeout  = 15 * time.Second
+		cmdCheck = "find /sys/devices/ -name %s" + console.CRLF
+	)
+
+	// example: /sys/devices/pci0000:00/0000:00:03.0/net/eth1`
+	ifacePath, err := console.RunCommandAndStoreOutput(vmi, fmt.Sprintf(cmdCheck, iface), timeout)
+	if err != nil {
+		return false, err
+	}
+	ifacePathSlice := strings.Split(ifacePath, "/")
+	if len(ifacePathSlice) < 4 {
+		return false, fmt.Errorf("Interface path %s is too show ", ifacePath)
+	}
+	var pciAddressCountAlongPath int
+	for _, token := range ifacePathSlice[3 : len(ifacePathSlice)-2] {
+		if isPCIAdsress(token) {
+			pciAddressCountAlongPath++
+		}
+	}
+	return pciAddressCountAlongPath == 1, nil
```

Comment:
For me, this is magic.
- How is a "root" represented in the guest? I think an overview (in the commit) is adequate to document this information.
- If the slice you get is 4, it means there is no token. I fail to understand the complexity of this logic if we are suppose to know in advance the exact format, including naming and slice size.